src/components/forms/FeedbackForm/FeedbackForm.tsx

```diff
@@ -20,14 +18,18 @@ interface FeedbackFormProps {
 }
 
 const options = [
-  { value: 'General', label: 'General' },
-  { value: 'Appearance', label: 'Appearance' },
-  { value: 'Accessibility', label: 'Accessibility' },
-  { value: 'Other', label: 'Other' },
+  { value: 'General', name: 'General' },
+  { value: 'Appearance', name: 'Appearance' },
+  { value: 'Accessibility', name: 'Accessibility' },
+  { value: 'Other', name: 'Other' },
 ];
 
+const topicOptions = options.map((option) => option.value);
+
 const FeedbackForm = ({ closeModal }: FeedbackFormProps) => {
   const [selectedValue, setSelectedValue] = useState('');
+  const [formSubmitted, setFormSubmitted] = useState(false);
```

Comment:
if i`m not mistaken there is a specific prop for isSubmitting form state in react hook form, but if it doesn't work, it's better to name it like "isFormSubmitting"